cfme/base/ssui_login.py

```diff
@@ -0,0 +1,49 @@
+from widgetastic.widget import View
```

Comment:
The contents of this file could go into the base/ssui.py file I believe. I don't think there is any reason for it not to. Please don't use the fact that we have a "login.py" file confuse the issue. Remember we are starting to move to Sentaku/New structure. The way I see things being handled in the future is like this,

Currently, we have
```a_model.py``` (could be host.py, doesn't matter) This file contains all the ui/db models, methods and pages that are related to this particular model.

We want to move that to something more akin to this
```
a_model/__init__.py     # The base model goes in here, the collection object also
a_model/ui.py           # Anything UI related, views/navigation etc
a_model/ssui.py         # Anything SSUI related
a_model/db.py           # Anything DB related
```

Does this structure make sense?